TODO/why-i-love-ugly-messy-interfaces-and-you-probably-do-too.md

```diff
@@ -1,62 +1,109 @@
+
+ 
+
 
 And that makes sense, really! Everyone likes easily digestible things that look bright and stylish. Nobody wants ugly, messy stuff.
+那真是有意义，真的！每个人都喜欢看起来亮丽时尚的那种容易理解消化的东西。没人想要丑的凌乱的食物。
+
 
 Or do they?
+是这样？
+
 
 #### Here’s some ugly design that’s unbeatable.
 
 ![](http://ww3.sinaimg.cn/large/a490147fgw1f2p9lj1q9zj20xc0qrtl1.jpg)
 
-#### Here’s some cluttered design that’s quite popular.
 
+###这里有一些丑到爆的设计。
+![](http://ww3.sinaimg.cn/large/a490147fgw1f2p9lj1q9zj20xc0qrtl1.jpg)
+
+
+
+#### Here’s some cluttered design that’s quite popular.
+####这里是很流行的一些杂乱的设计
 ![](http://ww1.sinaimg.cn/large/a490147fgw1f2p9lx308wj20xc0pqdka.jpg)
 
-#### Here’s some complex design that 1.5 billion people use every month.
 
+#### Here’s some complex design that 1.5 billion people use every month.
+####这是一些月用户一百五十万的复杂设计
 ![](http://ww1.sinaimg.cn/large/a490147fgw1f2p9m8c0rnj20xc0riqdh.jpg)
 
 So…wait. If beautiful, fresh, clean, and simple are so important, why hasn’t someone upended all of these products with something nicer? It’s not for a lack of trying. There are countless simpler, better-looking Craigslist and Photoshop competitors, for example.
+所以……等下。如果美丽、清新、干净、简单那么重要的话，为什么没人用更好的东西来颠覆上述所有这些产品呢？例如会出现有无数更简洁好看的的的克雷格网站和PS的竞争对手。
 
 The answer is that these products do an incredible job of solving their users’ problems, and their complex interfaces are a key reason for their success.
+问题的答案是，这些产品在解决用户问题方面做得极棒，而他们复杂的接口是其成功的关键。
 
 Let’s say your goal is to make a global peer-to-peer commerce network. That’s a big, complicated project to tackle.
+假设你的目的是完成一个全球的P2P商业网络。那真的需要解决一个巨大庞杂的项目。
 
 You could attempt to reduce your solution down to a minimal version, cutting out features and reducing density in the name of beauty and simplicity. Here’s a Craigslist redesign concept like that. (Designers sure hate Craigslist, don’t they? Has any other site had more unsolicited redesigns?)
 
 ![](http://ww4.sinaimg.cn/large/a490147fgw1f2p9mpbsv9j20m80dwdha.jpg)
 
+你会去尝试压缩你的解决方案到一个最简化的版本，以美丽简洁之名砍掉功能，减小密度。这里有一个如此的克雷格网站的设计理念。（设计者们恨透了克雷格，不是吗？其他的网站有这么多未经请求的重设计吗？）
+
 Or, you might decide that you really can’t cut features, because it’s more important to nail every use case you care about. (Remember, you have to support a huge number of scenarios to reach table stakes for this project.) Now beauty and simplicity are instantly a much lower priority. Making something useful comes first.
+或者，你决定不去砍掉功能，因为抓住你关注的每一个案例更重要。（记住，你必须支持大量的方案才能满足项目的投入需求。）现在美丽和简洁的优先级更低了。实用至上！
+
 
 For another example, think of Photoshop. How many graphic designers who idolize Swiss Style also use Photoshop every day? Probably most of them. Yet Photoshop’s UI is the antithesis of minimal — it has more nasty junk drawers than your parents’ unkempt basement. It doesn’t matter at all, because people don’t come to Photoshop for inspirational UI. They use it to get the job done.
+另一种情况，看看ps。有多少崇拜瑞士风格的设计师每天也的使用PS？ 或许其中多数都在用。然而PS的用户界面恰恰就不那么最简——它有着比家里凌乱地下室更恶心不堪的抽屉。
```

Comment:
It doesn’t matter at all, because people don’t come to Photoshop for inspirational UI. They use it to get the job done.
但这个并不重要，因为人们使用Photoshop并不是因为它拥有激动人心的UI，人们使用它是为了完成工作。